library-api/src/main/java/org/example/libraryApi/security/JwtRequest.java

```diff
@@ -0,0 +1,5 @@
+package org.example.libraryApi.security;
+
+public record JwtRequest(String username, String password) {
+}
```

Comment:
Для record такой формат записи выглядит лучше(имхо)
```suggestion
public record JwtRequest(
        String username,
        String password
) {
}
```